example/plopfile.js

```diff
@@ -134,14 +144,13 @@ module.exports = function (plop) {
 						return;
 					}
 				},
-				transform(fileContents, data) {
```

Comment:
unused?

Reply:
Let's leave as-is. This is meant as an example file, not as a complete code usage

Reply:
eslint shows it as error. Should we add ignore line?

Reply:
I think you can ignore the whole file. Like @crutchcorn said, it is an example file, not source code.